Update communication channels in documentation (#20429)
rapid7/metasploit-framework: docs/metasploit-framework.wiki/Contact.md

# Primary Communication Channels

## GitHub Discussions
For community support, questions, and general discussion, visit our [GitHub Discussions](https://github.com/rapid7/metasploit-framework/discussions).

## Slack
Join the [Metasploit Slack](https://join.slack.com/t/metasploit/shared_invite/zt-30i688it0-mJsFGT44IMtdeZi1DraamQ) for real-time chat with the community and developers.

## GitHub Issues
Submit bug reports and feature requests through [GitHub Issues](https://github.com/rapid7/metasploit-framework/issues).

# Additional Communication Channels

## Chat

Some community discussion still happens on IRC in #metasploit on Freenode.
Please be patient and hang around for a while -- not everyone is awake
at the same time as you. =)

## Mailing list

The Metasploit development mailing list used to be hosted on SourceForge, but is now on Google Groups. Metasploit Hackers is dead, long live [Metasploit Hackers][list]. (Or [mailto:Metasploit Hackers][mailto]).

The old list [is archived on seclists.org][archive].

## Social Media

> - **Twitter**: [@metasploit](https://twitter.com/metasploit)

Reviewer: ```suggestion
- **X**: [@metasploit](https://x.com/metasploit)
```

- **Mastodon**: [@metasploit@infosec.exchange](https://infosec.exchange/@metasploit)

> - **Blog**: [Rapid7 Blog - Metasploit Tag](https://www.rapid7.com/blog/tag/metasploit/)

Reviewer: ```suggestion
- **Blog**: [Rapid7 Blog - Metasploit Tag](https://www.rapid7.com/blog/tag/metasploit/)
- **YouTube**: [Metasploit YouTube](https://youtube.com/@MetasploitR7)
```

## Slack
Join the [Metasploit Slack](https://join.slack.com/t/metasploit/shared_invite/zt-30i688it0-mJsFGT44IMtdeZi1DraamQ) for real-time chat with the community and developers.

## GitHub Issues
Submit bug reports and feature requests through [GitHub Issues](https://github.com/rapid7/metasploit-framework/issues).

# Additional Communication Channels

## Chat

Some community discussion still happens on IRC in #metasploit on Freenode.
Please be patient and hang around for a while -- not everyone is awake
at the same time as you. =)

## Mailing list

The Metasploit development mailing list used to be hosted on SourceForge, but is now on Google Groups. Metasploit Hackers is dead, long live [Metasploit Hackers][list]. (Or [mailto:Metasploit Hackers][mailto]).

The old list [is archived on seclists.org][archive].

## Social Media

- **X**: [@metasploit](https://x.com/metasploit)
- **Mastodon**: [@metasploit@infosec.exchange](https://infosec.exchange/@metasploit)
- **Blog**: [Rapid7 Blog - Metasploit Tag](https://www.rapid7.com/blog/tag/metasploit/)
- **YouTube**: [Metasploit YouTube](https://youtube.com/@MetasploitR7)

# Abuse

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to msfdev@metasploit.com which goes to all the current committers. If the incident involves a committer, you may report directly to caitlin_condon@rapid7.com or todb@metasploit.com.

Reviewer 2: Caitlin is not part of R7, so maybe we should replace that? @zeroSteiner, would it make sense to add you there instead?

Reviewer 3: Yeah we should probably update it with my info, smcintyre@metasploit.com.

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to msfdev@metasploit.com which goes to all the current committers. If the incident involves a committer, you may report directly to caitlin_condon@rapid7.com or todb@metasploit.com.

Author: Committed!

## Slack
Join the [Metasploit Slack](https://join.slack.com/t/metasploit/shared_invite/zt-30i688it0-mJsFGT44IMtdeZi1DraamQ) for real-time chat with the community and developers.

## GitHub Issues
Submit bug reports and feature requests through [GitHub Issues](https://github.com/rapid7/metasploit-framework/issues).

# Additional Communication Channels

## Chat

Some community discussion still happens on IRC in #metasploit on Freenode.
Please be patient and hang around for a while -- not everyone is awake
at the same time as you. =)

## Mailing list

The Metasploit development mailing list used to be hosted on SourceForge, but is now on Google Groups. Metasploit Hackers is dead, long live [Metasploit Hackers][list]. (Or [mailto:Metasploit Hackers][mailto]).

The old list [is archived on seclists.org][archive].

## Social Media

- **X**: [@metasploit](https://x.com/metasploit)
- **Mastodon**: [@metasploit@infosec.exchange](https://infosec.exchange/@metasploit)
- **Blog**: [Rapid7 Blog - Metasploit Tag](https://www.rapid7.com/blog/tag/metasploit/)
- **YouTube**: [Metasploit YouTube](https://youtube.com/@MetasploitR7)

# Abuse

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to msfdev@metasploit.com which goes to all the current committers. If the incident involves a committer, you may report directly to smcintyre@metasploit.com or todb@metasploit.com.

Reviewer 2: Sorry about that, we might also need to update this address:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to msfdev@metasploit.com which goes to all the current committers. If the incident involves a committer, you may report directly to smcintyre@metasploit.com or jacquelyn_harris@rapid7.com.
```
